src/components/Single/Home/Home.vue

```diff
@@ -32,7 +32,7 @@
                     </div>
                     <div class="content">
                         We hebben een geweldige trip gehad naar het buitenland. <br>
-                        Zeker een aanrader.
+                        Zeker geen aanrander.
```

Comment:
please change back and if time to english

Reply:
This is just some demo content that will be removed as soon as it gets loaded from the database